src/qml/mixxxscreen.cpp

```diff
@@ -3,6 +3,37 @@
 namespace mixxx {
 namespace qml {
 
+void MixxxScreen::classBegin() {
+}
+
+void MixxxScreen::componentComplete() {
+    const auto* const meta = metaObject();
+    int typedIdx = meta->indexOfMethod(kScreenTransformFunctionTypedSignature);
+    int untypedIdx = meta->indexOfMethod(kScreenTransformFunctionUntypedSignature);
+    if (typedIdx >= 0) {
+        auto transformMethod = meta->method(typedIdx);
+        if (!transformMethod.isValid()) {
+            // TODO
```

Comment:
How do I get the engine from a QML element? I can see that [`QQuickView` has an `engine()`](https://doc.qt.io/qt-6.5/qquickview.html#engine) which I suppose is because [it takes the `QQmlEngine` has ctor argument](https://doc.qt.io/qt-6.5/qquickview.html#QQuickView-1) but I don't understand where it gets this argument from. In my code, if I try put any other ctor than `QObject`'s default, element stops being creatable.